### ISSUE TITLE
notification.presentNotification() not presenting notification?

### ISSUE THREAD

**Issue by dilizarov** [kunass2/BSForegroundNotification#1]
In my AppDelegate I have the following code:

```
func application(application: UIApplication, didReceiveRemoteNotification userInfo: [NSObject : AnyObject]) {

    let state = application.applicationState

    if state == .Active {
        let notification = BSForegroundNotification(userInfo: userInfo)
        notification.delegate = self
        notification.presentNotification()
    }
}
```

For some reason, the notification is not being presented. Is it not meant for use in AppDelegate?

**Comment by dilizarov** [comment]
Actually, I'm pretty sure it is because in userInfo, the aps dictionary can contain an alert that is a string or dictionary. You only handle when it is a dictionary, but don't handle when it is a string.

**Comment by dilizarov** [comment]
I was right in my assumption and included a fix in this pull request: #2

**Comment by kunass2** [comment]
To present notification, both: `title` and `body` of `alert` dictionary in `aps` dictionary cannot be empty. In case when you use the `alert` as `string` you just need to put there not empty string (after I merge your pull request). Notification can be presented from any place in the app.
